make HAVE_PTHREAD= fails

commit: 510a9834bc2312a1f6ecaaa3e27ac58375247d0e

command:

```
$ make HAVE_PTHREAD=
```

```
  CC      build-x86_64/src/audio.o
src/audio.c:1685:7: error: no member named 'thr' in 'struct aurx'
                rx->thr.start = rx->jbtype == JBUF_ADAPTIVE;
                ~~  ^
src/audio.c:2473:7: error: no member named 'thr' in 'struct aurx'
                rx->thr.start = rx->jbtype == JBUF_ADAPTIVE;
                ~~  ^
2 errors generated.
make: *** [build-x86_64/src/audio.o] Error 1
```

--- a/src/audio.c
+++ b/src/audio.c
@@ -1682,7 +1682,9 @@ static int start_player(struct aurx *rx, struct audio *a,
 		info("audio: player started with sample format %s\n",
 		     aufmt_name(rx->play_fmt));
 
+#ifdef HAVE_PTHREAD
 		rx->thr.start = rx->jbtype == JBUF_ADAPTIVE;
+#endif
 	}
 
 	return 0;
@@ -2469,8 +2471,9 @@ int audio_set_player(struct audio *a, const char *mod, const char *device)
 				mod, device, err);
 			return err;
 		}
-
+#ifdef HAVE_PTHREAD
 		rx->thr.start = rx->jbtype == JBUF_ADAPTIVE;
+#endif
 	}
 
 	return 0;

